prime-angular-frontend/src/app/modules/provision/shared/enums/enrolment-status.enum.ts

```diff
@@ -0,0 +1,8 @@
+export enum EnrolmentStatus {
```

Comment:
Are we going to grab this from lookups?

Reply:
We already do, but there is a need where the code for accepted or declined is required when updating a enrolment status.  The API takes a query parameter of status code, and I either hardcore it or use an enum.  The only other solution would be a more specific endpoint that is updateEnrolmentAccepted, updateEnrolmentDenied, etc instead of a single generic endpoint.